### PR TITLE
[LLM] support bloom dybatch

### PR DESCRIPTION
**Pull request by xiaoxiaohehe001**
### PR types

New features
### PR changes

Models
### Description

适配 bloom 细粒度

### CODE REVIEW

File: llm/predictor.py [hunk 1]
```diff
@@ -732,7 +760,7 @@ def predict():
                 target_texts.append(example["tgt"])
     else:
         source_texts = ["hello world, how are you?", "你好，请问你是谁?"]
-        target_texts = ["", ""]
+        target_texts = ["", "", ""]
```
Comment (wj-Mcat) [inline llm/predictor.py]: source_texts 的长度为 2， target_texts 的长度应该也是 2，所以这样代码删掉呗。

```suggestion
        target_texts = ["", ""]
```

File: paddlenlp/experimental/transformers/bloom/modeling.py [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+        hidden_states = transformer_outputs[0]
+        lm_logits = self.lm_head(hidden_states)
+
+        loss = None
+        if labels is not None:
+            # Shift so that tokens < n predict n
+            shift_logits = lm_logits[..., :-1, :]
+            shift_labels = labels[..., 1:]
+            # Flatten the tokens
+            loss = self.criterion(shift_logits, shift_labels)
```
Comment (wj-Mcat) [inline paddlenlp/experimental/transformers/bloom/modeling.py]: 把计算 loss 相关的代码给删掉吧，inference-model 是不存在这块的功能的。

**Review by wj-Mcat — CHANGES_REQUESTED** [review]

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Review by sijunhe — APPROVED** [review]

**Review by sijunhe — APPROVED** [review]
